### ISSUE TITLE
Support for the "pedal button" on some Wii guitars

### ISSUE THREAD

**Issue by 4ntsu** [madhephaestus/WiiChuck#4]
First of all, thanks for the amazing library!

Some Guitar Hero controllers have an RJ11 port at the bottom, which I believe was originally intended to be used with an effects pedal (that never got released lol). This port registers as a button when its contacts are shorted, and people have used this to build their own pedals, or in my case, wire it to a tilt switch to regain the Star Power functionality in Clone Hero (which would be provided by the gyroscopes in the Wiimote when the guitar is used with a Wii).

I'm using this library on a personal project, and I noticed it does not detect this button. With a Raphnet adapter, this is detected as button 9. I tried having a look at the code, but being just a hobbyist myself, implementing this seems beyond my skills.

Naturally, I'm happy to assist with testing if necessary.

Thanks.

**Comment by madhephaestus** [comment]
@acamilo  It looks like a new define needs to go here: https://github.com/madhephaestus/WiiChuck/blob/master/src/Guitar.h#L20 but I am at a loss as to what to put there? What is "button 9" in the context of the bytes? If you gave me the # define in the comments here, i would be happy to add it to the library.

**Comment by 4ntsu** [comment]
Hi,

Today I had another crack at this, and I think now I have a better understanding of how the I2C comms work.
I uncommented lines 316 and 319 in `Accessory.cpp` (thanks for leaving the debug stuff in!) and I got the following in the console:
Nothing pressed: `DATA=  , 31 , 33 , 15 , 16 , 255 , 255`
Pedal button pressed: `DATA=  , 31 , 33 , 15 , 16 , 255 , 251`

So it seems the change is in `WII_BYTE5` (6th byte), from `11111111` to `11111011` (WII_BIT2).

I have submitted a PR with the changes, tested and working on my controller.
